### PR TITLE
Initial amendments

### PR DESCRIPTION
**Pull request by asofmosq**

### CODE REVIEW

**Comment by asofmosq** [comment]
Retest this please

**Comment by mlaszloffy** [comment]
retest this please

**Comment by asofmosq** [comment]
I have the olive and niassawf ready to go (per Andre’s approval). I just need this PR approved, merged and installed in dev to see if we could perhaps get this in production before 1PM.

**Comment by iainrb** [comment]
Looks good so far. FYI I've run it successfully on a UGE cluster node using the Cromwell executable as follows:

`module load java/8 bbmap/38.75 bowtie2/2.3.5.1 samtools/1.9 hg38-bowtie-index/2.3.5.1 kraken2/2.0.8 kraken2-database/1 sars-covid-2-bowtie-index/2.3.5.1 ivar/1.0 bedtools bcftools/1.9 vcftools/0.1.16 seqtk/1.3 sars-covid-2/mn908947.3 blast spades/3.14.0
/.mounts/labs/gsi/modulator/sw/Ubuntu18.04/cromwell-45.1/bin/cromwell run --inputs inputs.json  ~/git/sarsCoV2Analysis/sarsCoV2Analysis.wdl
`

Where `inputs.json` is a copy of the WDL inputs from `tests.json`:

`{
    "sarsCoV2Analysis.fastq1": "/.mounts/labs/gsi/testdata/sarsCoV2Analysis/input_data/SRR11059944_1.fastq.gz",
    "sarsCoV2Analysis.fastq2": "/.mounts/labs/gsi/testdata/sarsCoV2Analysis/input_data/SRR11059944_2.fastq.gz",
    "sarsCoV2Analysis.samplePrefix": "SRR11059944",
    "sarsCoV2Analysis.primerBed": "/.mounts/labs/gsi/testdata/sarsCoV2Analysis/input_data/nCoV-2019.V2.bed",
    "sarsCoV2Analysis.ampliconBed": "/.mounts/labs/gsi/testdata/sarsCoV2Analysis/input_data/nCoV-2019.amplicons.V2.bed"
}
`

File: sarsCoV2Analysis.wdl [hunk 1]
```diff
@@ -0,0 +1,472 @@
...
+  String hostMappedBam_ = "~{sample}.host.mapped.bam"
+  String hostMappedBai_ = "~{sample}.host.mapped.bam.bai"
+
+  command <<<
+    set -euo pipefail
+
+    bowtie2 --quiet -x ~{reference} \
+    -1 ~{fastq1} -2 ~{fastq2} \
+    --un-conc-gz ~{sample}_host_removed_r%.fastq.gz \
+    -S ~{hostMappedSam}
```
Comment (iainrb) [inline sarsCoV2Analysis.wdl:212]: Could pipe `bowtie2` output directly into `samtools view -b` here -- faster and eliminates writing a SAM file to temporary space

File: sarsCoV2Analysis.wdl [hunk 2]
```diff
@@ -0,0 +1,472 @@
...
+    File genomecvgPerBase = "~{sample}.genome.cvgperbase.txt"
+    File hostMappedAlignmentStats = "~{sample}.host.mapped.samstats.txt"
+    File hostDepletedAlignmentStats = "~{sample}.samstats.txt"
+  }
+}
+
+task blast2ReferenceSequence {
+  input {
+    String modules = "blast"
+    String reference = "/.mounts/labs/gsiprojects/gsi/covid19/ref/MN908947.3.fasta"
```
Comment (iainrb) [inline sarsCoV2Analysis.wdl]: Better to have:
`String modules = "blast sars-covid-2/mn908947.3"`
`String reference = "$SARS_COVID_2_ROOT/MN908947.3.fasta"`

**Review by morgantaschuk — APPROVED** [review]

**Review by mlaszloffy — APPROVED** [review]

**Review by iainrb — APPROVED** [review]

**Pull request merged**